assets/Template.hx

```diff
@@ -0,0 +1,9 @@
+class Main {
+    static function main() {
+		var sample = "My name is <strong>::name::</strong>, <em>::age::</em> years old";
+		var user = {name:"Mark", age:30};
+		var template = new haxe.Template(sample);
```

Comment:
indent

In general please use two-space indents for code samples. I realize that's not typical Haxe indenting but we have to be careful with the width of code samples.